### DISCUSSION TITLE
Please porting this emulator to nintendo 3ds or nintendo switch

### DISCUSSION THREAD

**Discussion by BanhMi939** [SquirrelJME/SquirrelJME#148] [RetroArch (RatufaCoat)] [answered]
First, thanks you for your hard work on this awsome project, as a guy who spent my entire childhood played j2me game on old nokia, could i suggest you to port your emulator to nintendo 3ds or nintendo switch for every old school gamers can enjoy those games again? Sorry for my bad english but i really love if it may come to 3ds or switch.

**Comment by XerTheSquirrel** [comment, marked as answer]
Hi! When this is ready, there will be RetroArch cores that will run on 3DS and Switch: Typically any platform supported by RetroArch: <https://www.retroarch.com/?page=platforms>.
